misc/netgen/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG, GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: netgen
+  version: {{ version }}
+
+source:
+   git_url: https://github.com/RTimothyEdwards/netgen
+   git_rev: master
+
+build:
+  # number: 201803050325
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20180305_0325
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+    - TRAVIS
+    - CONDA_OUT
+    - TEST_PACKAGE
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - tcsh
+    - cairo
+    - m4
+    - tk
+    - ncurses
+    - python 3.7
```

Comment:
As reported by GLatosinski regarding the magic package, this should probably be `python>=3.7`. Skywater PDK relies on Python 3.8.

Reply:
Fixed.